disambiguate the origin (#17)
infracost/jetbrains-infracost: README.md

## Description

<!-- Plugin description -->

> Infracost is an IntelliJ-based plugin that allows you to shift left on your Cloud costs by providing cost estimates for

Reviewer: Maybe:
```suggestion
Infracost is a plugin for IntelliJ IDEs that allows you to shift left on your Cloud costs by providing cost estimates for
```
I'm good either way
